File: microservices/delivery-order-service/src/main/java/com/server/deliveryorderservice/model/entity/OrderItem.java
package com.server.deliveryorderservice.model.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderItem {

    private Long id;
    private String name;
    private String description;
    private String notes;
    private Integer quantity;
    private Double price;
    private Double promotionalPrice;
    private Double totalPrice;
    private Double totalPriceWithDiscount;

}

File: web/customer-app/src/app/models/order/order-item.ts
export class OrderItem {

  constructor(quantity: number, menuItemId: number) {
    this.quantity = quantity;
    this.menuItemId = menuItemId;
  }

  quantity: number;
  menuItemId: number;
  price: number;
  promotionalPrice: number;
  name: string;
}
